src/Emogrifier.php

```diff
@@ -375,7 +375,27 @@ protected function process(\DOMDocument $xmlDocument)
             $allCss .= $this->getCssFromAllStyleNodes($xPath);
         }
 
-        $cssParts = $this->splitCssAndMediaQuery($allCss);
+        $splitCss = $this->splitCssAndMediaQuery($allCss);
+        // this is temporary pending refactoring `parseCssRules` and `copyCssWithMediaToStyleNode`
```

Comment:
Even if the code is temporary, please let's move it into a well-named private method instead of making this existing method even longer.